src/index.html

```diff
@@ -6,13 +6,68 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <title>Product cards</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="imac image"
+      />
+
+      <div class="card__text-container">
```

Comment:
Do you need this container? It's redundant
```suggestion
```

Reply:
It seemed to me easier to create a container and set margins for it so that they are set for all elements rather than adding them to each one individually. Isn't that right?